examples/st-device-app/android/App/core/common/src/main/java/com/samsung/matter/chipstdeviceapp/core/common/DeepLink.kt

```diff
@@ -0,0 +1,22 @@
+package com.samsung.matter.chipstdeviceapp.core.common
+
+import androidx.core.net.toUri
+import androidx.navigation.NavDeepLinkRequest
+
+object DeepLink {
+  fun getDeepLinkRequestForQrcodeFragment(setting: String): NavDeepLinkRequest {
+    return NavDeepLinkRequest.Builder.fromUri(
+        "android-app://com.samsung.matter.chipstdeviceapp.feature.qrcode/qrcodeFragment/${setting}"
```

Comment:
why are the paths `com.samsung`? If this is to become an SDK supported thing, should this be `com.csa` or `com.matter` or similar?

Reply:
We changed it to "com.matter" same as other examples

Reply:
Thank you for your feedback. We changed name space to "com.matter" same as other examples